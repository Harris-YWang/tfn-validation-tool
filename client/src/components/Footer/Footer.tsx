import React from 'react';
import './footer.css';

const Footer = () => (
	<footer className="footer">
		<div className="container">
			<span>
				Copyright Â© {new Date().getFullYear()} by{' '}
				<a href="https://github.com/Harris-YWang">Harris Wang</a>
				{'.'}
			</span>
		</div>
	</footer>
);

export default Footer;
