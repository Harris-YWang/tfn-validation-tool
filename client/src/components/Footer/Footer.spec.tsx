import { shallow } from 'enzyme';
import React from 'react';
import Footer from './index';

describe('Footer Component', () => {
	it('should render without crashing', () => {
		const wrapper = shallow(<Footer />);

		expect(wrapper).toMatchSnapshot();
	});
});
