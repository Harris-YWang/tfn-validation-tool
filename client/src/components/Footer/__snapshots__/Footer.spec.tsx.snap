// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component should render without crashing 1`] = `ShallowWrapper {}`;
